src/Tasks/system.design/stronglytypedresourcebuilder.cs

```diff
@@ -296,6 +335,9 @@ internal static CodeCompileUnit Create(String resxFile, String baseName, String
             // keywords, etc.  So there's no point to duplicating the code above.
 
             return InternalCreate(resourceList, baseName, generatedCodeNamespace, resourcesNamespace, codeProvider, internalClass, out unmatchable);
+#else
+            throw new NotImplementedException();
```

Comment:
This should probably have a better error message describing what isn't supported.

Reply:
I don't know about MSBuild usage, but in CoreFX `NotImplementedException` is used when something is not implemented by design (such as, it needs to be overridden) and `PlatformNotSupportedException` when it's not relevant to the platform or not yet implemented there.